src/nodeconductor_test/azure_instance_creation_test.py

```diff
@@ -0,0 +1,180 @@
+"""
+1. Login NC
+2. Choose organization
+3. Create project
+4. Create provider
+5. Create resource
+6. Delete resource
+7. Delete provider
+8. Delete project
+"""
+
+
+import time
+import unittest
+
+from helpers import (login_nodeconductor, get_driver, create_project, delete_project, create_provider_azure,
+                    choose_organization, create_resource_azure, element_exists, delete_resource, is_in_list,
+                    delete_provider)
+
+
+class Settings(object):
+    site_url = "http://web-test.nodeconductor.com"
+    username = 'Alice'
+    password = 'Alice'
+    user_full_name = 'Alice Lebowski'
+    nec_organization = 'Ministry of Bells'
+    project_name = 'xcghh'
+    provider_name = 'AzureTest provider'
+    provider_type_name = 'azure'
+    subscription_id_name = '39e917e7-d6d4-4a01-ac7c-1ce78a63013f'
+    category_name = 'VMs'
+    os_password = 'Zi_lT6V0C9-~'
+    resource_name = 'AzureTest'
+    image_name = 'Visual Studio Community 2013 Update 4 with Tools for Node.js on Windows Server 2012 R2'
+    size_name = 'Medium Instance'
+    time_after_resource_creation = 180
+    time_wait_after_resource_stopping = 60
+    time_wait_after_resource_removal = 30
+
+
+class NodeconductorTest(unittest.TestCase):
```

Comment:
1. Please add some explanation that this test is not ready because its impossible to test resource deletion
2. Please change test class name to `AzureResourceCreationTest`